alleycats-laws/src/main/scala/alleycats/laws/discipline/FlatMapRecTests.scala

```diff
@@ -0,0 +1,29 @@
+package alleycats.laws.discipline
+
+import cats._
+import cats.laws.FlatMapLaws
+import cats.laws.discipline._
+import org.scalacheck.Arbitrary
+import org.scalacheck.Prop._
+import org.typelevel.discipline.Laws
+
+
+trait FlatMapRecTests[F[_]] extends Laws {
```

Comment:
this looks duplicative.

Reply:
I believe this is the only way to test the one specific rule out of the rule set, in this case `tailRecMConsistentFlatMap`